Fix is on page method

wellavelino opened the pull request:
This method had a boolean context, but not retun true or false
im change to a void method now the name is is_on_page!

cs-wellington-santos committed:
Fix is on page method

- This method had a boolean context, but not retun true or false
im change to a void method now the name is is_on_page!

oscartanner commented:
@wellavelino Don't you think that would be nice to have the `is_on_page?` method?

Just do:

```
def is_on_page?(page_text, negation = '')
  begin
    is_on_page!(page_text, negation)
  rescue
    return false
  end
  true
end
```

oscartanner commented:
@wellavelino Don't forget to update all project files to use the new function `is_on_page!`.

One example [here](https://github.com/concretesolutions/sunomono/blob/master/lib/templates/base_steps.tt#L7)

cs-wellington-santos committed:
Adding is_on_page? method

Adding boolean is_on_page?, always returns true or false
instead exception

cs-wellington-santos committed:
Change Ioscreen Base

- Adding is_on_page? on iosscrenbase
- Remove is_on_page? method from base steps
and include is_on_page!

oscartanner commented on lib/templates/android_screen_base.tt:
```diff
@@ -125,6 +125,19 @@ class AndroidScreenBase < Calabash::ABase
   def is_on_page?(page_text, negation = '')
   fail 'Error! Invalid query string!' if
```
Indentation problem?

oscartanner commented on lib/templates/android_screen_base.tt:
```diff
@@ -125,6 +125,19 @@ class AndroidScreenBase < Calabash::ABase
+  # Negation indicates that we want a page that doesn't
+  # has the message passed as parameter
+  def is_on_page!(page_text, negation = '')
+  fail 'Error! Invalid query string!' if
```
Indentation problem?

oscartanner commented on lib/templates/ios_screen_base.tt:
```diff
@@ -142,9 +142,22 @@ class IOSScreenBase < Calabash::IBase
+  # Negation indicates that we want a page that doesn't
+  # has the message passed as parameter
+  def is_on_page?(page_text, negation = '')
+  fail 'Error! Invalid query string!' if
```
Indentation problem?

oscartanner commented on lib/templates/ios_screen_base.tt:
```diff
@@ -142,9 +142,22 @@ class IOSScreenBase < Calabash::IBase
-  def is_on_page?(page_text, negation = '')
+  def is_on_page!(page_text, negation = '')
   fail 'Error! Invalid query string!' if
        page_text.to_s == ''
```
Indentation problem?

oscartanner reviewed (commented):
Fine for me, just 4 very little fixes :)

cs-wellington-santos committed:
Adjust identation problem on screen bases

cs-wellington-santos committed:
Adjust identation problem

oscartanner reviewed (approved):
👍
